resources/views/admin/categories/create-parent.blade.php

```diff
@@ -0,0 +1,22 @@
+@extends('shop.shop_layouts.master')
```

Comment:
@afnanali1992 Why extending front-end layout? Please make it extending the backend.